cli/cli.go

```diff
@@ -29,6 +29,7 @@ type UserOption struct {
 	Latest               bool                // used in conjunction with --restart to check for latest version of service(s) being restarted
 	List                 bool                // lists all the services
 	Logs                 string              // prints the logs of a service, running or otherwise
+	NoPortCheck          bool                // stops the `lsof` port check - used by jenkins
```

Comment:
Can we remove `used by Jenkins` - I don't think we should be documenting that here